### DISCUSSION TITLE
1.播放器界面进度条太细，不利于鼠标操作。2.希望加入IINA播放器那种双指触摸板横向滑动调整播放进度功能！

### DISCUSSION THREAD

**Discussion by usercging** [hooke007/mpv_PlayKit#452] [libmpv]
如题，开发者你好，两点小建议：
1.目前OSD界面播放器下方进度条太细，鼠标每次需要找一下位置才能精准放到进度条细线上，希望能加粗一点（方便鼠标落上去），可改成办透明进度条（不遮挡画面。）
2.mac os 上面最棒的播放器是IINA，这个播放器有个功能就是双指触摸板横向滑动的时候可以直接调整播放进度（不需要拖动进度条即可），希望mpv-lazy能加上这个功能，因为很多MPV很多用户是笔记本，这样一个桌面端的mpv-lazy就有接近手机端播放器的拖动体验了，真的很棒的体验，希望开发者可以加入！！！！

**Comment by hooke007** [comment]
1你确定没发错区？
2截图给我看看多细，脚本选项能控制缩放和宽度
3我没有触控需求

**Comment by zhongfly** [comment]
这里也不是mpv开发仓库，只是配置文件分享…功能类请求请去 https://github.com/mpv-player/mpv

调整设置请自己去看配置文件，都有注释的，这里不是ChatGPT。

多点触控，mpv目前没有实现。触摸板是否有额外支持也不知道，我猜没有。具体请看mpv官方手册 https://mpv.io/manual/master/

**Comment by usercging** [comment]
开发者你好，解决了，mpv-lazy的OSD进度条是宽的，因为我电脑上有旧版mpv-net CM，播放的时候默认用mpvCM了，没有调用成功mpv-lazy，所以看到的osd进度条还是旧的，手动改为mpv-lazy，进度条很宽，够用了。谢谢回复。


发件人: Karl Hook ***@***.***>
发送时间: 2024年4月29日 15:58
收件人: hooke007/MPV_lazy ***@***.***>
抄送: usercging ***@***.***>; Author ***@***.***>
主题: Re: [hooke007/MPV_lazy] 1.播放器界面进度条太细，不利于鼠标操作。2.希望加入IINA播放器那种双指触摸板横向滑动调整播放进度功能！ (Discussion #452)


1你确定没发错区？
2截图给我看看多细，脚本选项能控制缩放和宽度
3我没有触控需求

**Comment by usercging** [comment]
谢谢，发帖位置选错了

发件人: zhongfly ***@***.***>
发送时间: 2024年4月29日 16:17
收件人: hooke007/MPV_lazy ***@***.***>
抄送: usercging ***@***.***>; Author ***@***.***>
主题: Re: [hooke007/MPV_lazy] 1.播放器界面进度条太细，不利于鼠标操作。2.希望加入IINA播放器那种双指触摸板横向滑动调整播放进度功能！ (Discussion #452)


这里也不是mpv开发仓库，只是配置文件分享…

调整设置请自己去看配置文件，都有注释的，这里不是ChatGPT。

多点触控，mpv目前没有实现
